src/migration/1702045361925-NotificationV2.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class NotificationV21702045361925 implements MigrationInterface {
+    name = 'NotificationV21702045361925'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Missing replica identity full for new table?

Reply:
Good catch!